apps/src/studio/levels.js

```diff
@@ -766,10 +764,17 @@ levels.iceage_6 = utils.extend(levels.playlab_6, {
   firstSpriteIndex: 3, // diego
   goalOverride: {} // This prevents the override from original playlab from being used
 });
-levels.gumball_6 = utils.extend(levels.playlab_6, {
+
+levels.gumball_6 = utils.extend(levels.playlab_5, {
```

Comment:
I'm a big fan of naming levels instead of using numbers, and now that we have this odd numbering (6 inheriting from 5) it seems like a good time to ponder that again.